docs/guide/styles.md

```diff
@@ -1,15 +1,320 @@
+In this chapter will explore how you can apply styles to your application to create beautiful user interfaces.
 
-- What are styles
-- Styles object on widgets / app
-- Setting styles via CSS
-- Box model
-- Color / Background
-- Borders / Outline
 
+## Styles object
+
+Every Textual widget class provides a `styles` object which contains a number of attributes. These attributes tell Textual how the widget should be displayed. Setting any of these attributes will update the screen accordingly.
+
+!!! note
+
+    These docs use the term *screen* to describe the contents of the terminal, which will typically be a window on your desktop.
+
+Let's look at a simple example which sets styles on `screen` (a special widget that represents the screen).
+
+```python title="screen.py" hl_lines="6-7"
+--8<-- "docs/examples/guide/styles/screen.py"
+```
+
+The first line sets the [background](../styles/background.md) style to `"darkblue"` which will change the background color to dark blue. There are a few other ways of setting color which we will explore later.
+
+The second line sets [border](../styles/border.md) to a tuple of `("heavy", "white")` which tells Textual to draw a white border with a style of `"heavy"`. Running this code will show the following:
+
+```{.textual path="docs/examples/guide/styles/screen.py"}
+```
+
+## Styling widgets
+
+Setting styles on screen is useful, but to create most user interfaces we will also need to apply styles to other widgets.
+
+The following example adds a static widget which we will apply some styles to:
+
+```python title="widget.py" hl_lines="7 11-12"
+--8<-- "docs/examples/guide/styles/widget.py"
+```
+
+The compose method stores a reference to the widget before yielding it. In the mount handler we use that reference to set the same styles on the widget as we did for the screen example. Here is the result:
+
+```{.textual path="docs/examples/guide/styles/widget.py"}
+```
+
+Widgets will occupy the full width of the screen and as many lines as required to fit in the vertical direction.
+
+Note how the combined height of the widget is three rows in the terminal. This is because a border adds two rows (and two columns). If you were to remove the line that sets the border style, the widget would occupy a single row.
+
+Widgets will wrap text by default. If you were to replace `"Textual"` with a long paragraph of text, the widget will expand downwards to fit.
+
+## Colors
+
+There are a number of style attribute which accept colors. The most commonly used are [color](../styles/color.md) which sets the default color of text on a widget, and [background](..styles/background/md) which sets the background color (beneath the text).
+
+You can set a color value to one of a number of pre-defined color constants, such as "crimson", "lime", and "palegreen". You can find a full list in the [Color reference](../reference/color.md#textual.color--named-colors).
+
+Here's how you would set the screen background to lime:
```

Comment:
The text says "screen", and the example shows an arbitrary widget, suggesting:

```suggestion
Here's how you would set the background of a widget to lime:
```

Reply:
Updated the example. Wanted to stick to screen for now.